src/main/java/gov/cms/mat/cql_elm_translation/service/filters/CqlTranslatorExceptionFilter.java

```diff
@@ -39,7 +40,17 @@ public List<CqlCompilerException> filter() {
       if (filteredCqlTranslatorExceptions.isEmpty()) {
         return Collections.emptyList();
       } else {
```

Comment:
`else` statement can be safely removed. The `return` statement will prevent the current `else` block from executing.

Reply:
@jkotanchik-SB Thanks a lot Joe! Just did the changes.